mh/pat.py

```diff
@@ -256,9 +256,16 @@ def recvAnsConnection(self, packet_id, data, seq):
 
         The games sends the PAT environment properties.
         """
-        connection_data = pati.ConnectionData.unpack(data)
-        self.server.debug("Connection: {!r}".format(connection_data))
-        self.sendNtcLogin(5, connection_data, seq)
+        settings = pati.ConnectionData.unpack(data)
+        self.server.debug("Connection: {!r}".format(settings))
+         pat_ticket = b""
```

Comment:
```suggestion
        pat_ticket = b""
```
I'm having `IndentationError: unexpected indent` error.